Simplenote/src/main/java/com/automattic/simplenote/NoteListFragment.java

```diff
@@ -140,6 +148,7 @@ public void onNoteSelected(String noteID, int position, String matchOffsets, boo
     private TextView mSortOrder;
     private refreshListTask mRefreshListTask;
     private refreshListForSearchTask mRefreshListForSearchTask;
+    private int mIndex;
```

Comment:
is this name a little generic for the size of this class?
what are the reasons for declaring here instead of as a local variable in `deleteSearchItem` and in the delete button callback?

Reply:
> it can't be a local variable

ahh…yeah I figured that out later and then forgot to come back and delete this comment or acknowledge that I figured it out. thanks